### ISSUE TITLE
"You must Install or Update .Net to run this application"

### ISSUE THREAD

**Issue by rtizzy** [zaimoni/RSRevived#17]
Replication:

1. Download this [version](https://github.com/zaimoni/RSRevived/releases/download/unstable-2022-11-08/RSRevived.0.10.0-unstable-20221109.zip).
2. Attempt to run and see the error displayed in the screenshot
3. DOwnload and install .net using the "yes" option
4. Restart the computer to be sure
5. Try to run again and see the same error. 

![image](https://user-images.githubusercontent.com/523926/212738213-f3b403ed-81f7-4a35-bee8-901fb2967618.png)

**Comment by rtizzy** [comment]
I'd add appropriate labels but it doesn't appear to be possible for me to do so.

**Comment by zaimoni** [comment]
Right...interesting (this build was intended to be *before* I downloaded/installed .NET 7.0).  Does searching the Microsoft website give the same download-from page as the Yes button?

**Comment by rtizzy** [comment]
@zaimoni 

I found the problem. The application seems to point to the `Runtime`. The `Desktop Runtime` is what is actually required. After installing this I am able to start the game. 

I was prompted to download [this](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-6.0.13-windows-x64-installer)

What seems to be needed is [this](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.13-windows-x64-installer)
